src/features/calendar-events/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { Calendar, momentLocalizer } from 'react-big-calendar'
+import moment from 'moment'
+
+import events from './tempEvents'
+
+import 'react-big-calendar/lib/css/react-big-calendar.css'
+
+const localizer = momentLocalizer(moment)
+
+const CalendarRBC = () => (
+  <div className="calendar-rbc-container">
+    <Calendar
+      defaultDate={new Date()}
+      drilldownView="agenda"
+      defaultView="month"
+      events={events}
+      localizer={localizer}
+      resizable
+      style={{ padding: 15, width: '90%', margin: '0 auto', height: '85vh' }}
+    />
```

Comment:
I guess that more practical will be to have a separate file such as 'calendar-config.js' with all calendar properties